interfaces/api/display/I3DPointsViewer.h

```diff
@@ -60,19 +60,34 @@ class XPCF_CLIENTUUID("423cd9a6-f93a-4e1b-88f0-0762c1db5db3") XPCF_SERVERUUID("1
     /// @param[in] keyframesPoses (optional), poses of a set of keyframes (transform of the camera defined in world corrdinate system).
     /// @param[in] framePoses (optional), poses of a set of frames (transform of the camera defined in world corrdinate system).
-    /// @param[in] points2 (optional), a second set of 3D points to display in the 3D viewer (useful to visualize result of a bundle adjustment).
+    /// @param[in] pointCloud2 (optional), a second point cloud to display in the 3D viewer (useful to visualize result of a bundle adjustment).
     /// @param[in] keyframesPoses2 (optional), a second set of keyframes poses (transform of the camera defined in world corrdinate system, useful to visualize result of a bundle adjustment).
     /// @return FrameworkReturnCode::_SUCCESS if the window is created, else FrameworkReturnCode::_ERROR_
-    virtual FrameworkReturnCode display(const SRef<SolAR::datastructure::PointCloud> points,
+    virtual FrameworkReturnCode display(const SRef<SolAR::datastructure::PointCloud> pointCloud,
                                         const SolAR::datastructure::Transform3Df & pose,
                                         const std::vector<SolAR::datastructure::Transform3Df> & keyframePoses = {},
                                         const std::vector<SolAR::datastructure::Transform3Df> & framePoses = {},
-                                        const SRef<SolAR::datastructure::PointCloud> points2 = nullptr,
+                                        const SRef<SolAR::datastructure::PointCloud> pointCloud2 = nullptr,
                                         const std::vector<SolAR::datastructure::Transform3Df> & keyframePoses2 = {}) = 0;
+
+    /// @brief Add in the GPU buffer one, or optionnally two 3D point clouds. Required when these point clouds are huge an static.
+    /// @param[in] points set of static 3D points to display in the 3D viewer.
+    /// @param[in] points2 (optional), a second set of static 3D points to display in the 3D viewer (useful to visualize result of a bundle adjustment).
+    /// @return FrameworkReturnCode::_SUCCESS if the window is created, else FrameworkReturnCode::_ERROR_
+    virtual FrameworkReturnCode buffer(const std::vector<SRef<SolAR::datastructure::CloudPoint>> & points,
+                                        const std::vector<SRef<SolAR::datastructure::CloudPoint>> & points2 = {}) = 0;
+
+
+    /// @brief Add in the GPU buffer one, or optionnally two, 3D point clouds. Required when these point clouds are huge an static.
+    /// @param[in] pointCloud a static point cloud to display in the 3D viewer.
+    /// @param[in] pointCloud2 (optional), a second static point cloud to display in the 3D viewer (useful to visualize result of a bundle adjustment).
+    /// @return FrameworkReturnCode::_SUCCESS if the window is created, else FrameworkReturnCode::_ERROR_
+    virtual FrameworkReturnCode buffer(const SRef<SolAR::datastructure::PointCloud> pointCloud,
```

Comment:
`const std::vector<SRef<const SolAR::datastructure::PointCloud>> &` ?

Reply:
I prefer to address this point of missing const in SRef not only for this interface, but for all intercaes of the SolAR Framework.